dv-export/python/query-changes/queries/applicant.json

```diff
@@ -0,0 +1,66 @@
+{
+  "source": {
+    "analyticObject": "Applicant"
+  },
+  "columns": [
+    {
+      "columnName": "ApplicantID",
+      "columnDefinition": {
+        "property": {
+          "name": "Applicant.ApplicantID",
+          "qualifyingPath": "Applicant"
+        }
+      }
+    },
+    {
+      "columnName": "Validity_Start",
+      "columnDefinition": {
+        "property": {
+          "name": "Applicant.ValidityStart",
+          "qualifyingPath": "Applicant"
+        }
+      }
+    },
+    {
+      "columnName": "Validity_End",
+      "columnDefinition": {
+        "property": {
+          "name": "Applicant.ValidityEnd",
+          "qualifyingPath": "Applicant"
+        }
+      }
+    },
+    {
+      "columnName": "Offer_Salary_Amount",
+      "columnDefinition": {
+        "property": {
+          "name": "Applicant.Offer_Salary_Amount",
+          "qualifyingPath": "Applicant"
+        }
+      }
+    }
+  ],
+  "sortOptions": [
+    {
+      "columnIndex": 1,
+      "sortDirection": "SORT_ASCENDING"
+    }
+  ],
+  "filters": [
+    {
+      "memberSet": {
+        "dimension": {
+          "name": "ApplicantID_Hierarchy",
+          "qualifyingPath": "Applicant"
+        },
+        "values": {
+          "included": "${{ApplicantID}}"
+        }
+      }
+    }
+  ],
+  "options": {
+    "recordMode": "CHANGES",
+    "limit": 10000
```

Comment:
Don't think we should/need to limit this (in any of the queries)

Reply:
Yes, we should. There is a default_value for that in ChangesFetcher.DEFAULT_QUERY_LIMIT = 1000. If the limit is not set, ChangesFetcher will use the default one.